src/core/markdown.js

```diff
@@ -273,6 +273,27 @@ function structure(fragment, doc) {
   return process(fragment);
 }
 
+/**
+ * Re-structure DOM around elem whose markdown has been processed.
+ * @param {Element} elem
+ */
+function restructure(elem) {
+  const structuredInternals = structure(elem, elem.ownerDocument);
+  elem.setAttribute("aria-busy", "true");
```

Comment:
I realize now that this doesn't make any sense, because these are synchronously processed. Same with the one below.